ibis/expr/types/relations.py

```diff
@@ -240,21 +240,39 @@ def _fast_bind(self, *args, **kwargs):
                 args = ()
             else:
                 args = util.promote_list(args[0])
-        # bind positional arguments
+
         values = []
+        errs = []
+        # bind positional arguments
         for arg in args:
```

Comment:
This is somewhat orthogonal to the FieldNotFoundError stuff. But if I have multiple bogus columns, eg `table.select("bogus", "also_bogus")` I only see the first error.